tests/integration/pilot/analyze_test.go

```diff
@@ -119,15 +120,15 @@ func TestDirectoryWithRecursion(t *testing.T) {
 	framework.
 		NewTest(t).
 		RequiresSingleCluster().
-		Run(func(ctx framework.TestContext) {
+		Run(func(t framework.TestContext) {
 			g := NewWithT(t)
 
-			ns := namespace.NewOrFail(t, ctx, namespace.Config{
+			ns := namespace.NewOrFail(t, t, namespace.Config{
 				Prefix: "istioctl-analyze",
 				Inject: true,
 			})
 
-			istioCtl := istioctl.NewOrFail(ctx, ctx, istioctl.Config{})
+			istioCtl := istioctl.NewOrFail(t, t, istioctl.Config{})
```

Comment:
we may just make this only take in a resource.Context potentially. Probably a future PR